src/einsteinpy/utils/coord_transforms.py

```diff
@@ -1,219 +0,0 @@
-import warnings
```

Comment:
same question here ;)

Reply:
Also, tests need to be ported.

Reply:
what should I do exactly? for tests

Reply:
see `einsteinpy/testing` for this :) 
You would get the idea!